src/sidebar/components/annotation-header.js

```diff
@@ -47,11 +51,8 @@ export default function AnnotationHeader({
   const annotationIsPrivate = isPrivate(annotation.permissions);
   const annotationLink = annotation.links ? annotation.links.html : '';
 
-  // NB: `created` and `updated` are strings, not `Date`s
-  const hasBeenEdited =
-    annotation.updated && annotation.created !== annotation.updated;
   const showTimestamp = !isEditing && annotation.created;
-  const showEditedTimestamp = hasBeenEdited && !isCollapsedReply;
+  const showEditedTimestamp = hasBeenEdited(annotation) && !isCollapsedReply;
```

Comment:
Since the call to `hasBeenEdited` may involve possibly-expensive date string parsing, I think it would be worth caching with `useMemo(...)` here.